pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchFromGitLab , which, go-md2man, coreutils}:
+
+stdenv.mkDerivation rec {
+  version = "1.4.3";
+  name = "brillo-${version}";
+  src = fetchFromGitLab {
+    owner= "cameronnemo";
+    repo= "brillo";
+    rev= "v${version}";
+    sha256 = "1syv3iav7bwr84x9frz1qd6qmgp8ldbjs4gl3r94nhllkai9spaq";
+  };
+  makeFlags = [ "PREFIX=" "DESTDIR=$(out)"];
+  nativeBuildInputs = [go-md2man which];
+  buildFlags = [ "dist" ];
+  installTargets = "install-dist";
+
+  postPatch = ''
+    substituteInPlace contrib/90-brillo.rules --replace /bin/ ${coreutils}/bin/
```

Comment:
Can you use patch instead? This will break silently when the rules file is changed.

Reply:
I tried to stay very similar to the brightnessctl package, should this package be changed as well if I change this to use a patch instead of a command?

Reply:
Yeah, we have a ton of `substitute`s that do not do anything in the tree and we should work on eliminating them. There is no hurry, though.

Reply:
How can you make a patch with the coreutils location in?

Reply:
You create patch containing `@coreutils@` and use `substituteAll` to replace it.

Reply:
I've got something working but it doesn't look correct

Reply:
Here is an example: https://github.com/NixOS/nixpkgs/blob/749c0a9c16e4ab6cee82884e949eb9fac93f6261/pkgs/applications/backup/deja-dup/default.nix#L20-L24

Reply:
ok I'll try that